pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -396,6 +392,7 @@ func getTPMapMatchingExistingAntiAffinity(pod *v1.Pod, nodeInfoMap map[string]*s
 		node := nodeInfo.Node()
 		if node == nil {
 			catchError(fmt.Errorf("node not found"))
+			cancel()
```

Comment:
The cancel() operation does not seem to be related to the PR.

Reply:
If this is a issue, I suggest a separate PR to complete in order to better track.

Reply:
ok

Reply:
> The cancel() operation does not seem to be related to the PR.

done

Reply:
> If this is a issue, I suggest a separate PR to complete in order to better track.

I had opened a new issue for this, ref: https://github.com/kubernetes/kubernetes/pull/79774.